### PR TITLE
Fix/aria label is missing

### PR DESCRIPTION
**Pull request by marwan-mohamed12**
For Details See #6414 
- added the aria-label for the RSS link
- added the translation key for RSS link in en.json

### CODE REVIEW

File: components/Blog/BlogHeader/index.tsx [hunk 1]
```diff
@@ -18,12 +18,13 @@ const BlogHeader: FC<BlogHeaderProps> = ({ category }) => {
...
     siteConfig.rssFeeds.find(item => item.category === category)?.file ??
     'blog.xml';
+  const ariaLabel = t('layouts.blog.blogHeader.RSS');
 
   return (
     <header className={styles.blogHeader}>
       <h1>
         {t('layouts.blog.title')}
-        <Link href={`/feed/${currentFile}`}>
+        <Link href={`/feed/${currentFile}`} aria-label={ariaLabel}>
```
Comment (AugustinMauroy) [inline components/Blog/BlogHeader/index.tsx]:
```suggestion
        <Link href={`/feed/${currentFile}`} aria-label={ t('layouts.blog.blogHeader.RSS')}>
```

You can do that like for subtitle

Comment (marwan-mohamed12) [reply components/Blog/BlogHeader/index.tsx]: yeah, I know but  I did it like the referenced  link  
like [here](https://github.com/nodejs/nodejs.org/blob/004c16e039dbda73ab5048a607118ae27acca29f/components/Common/ThemeToggle/index.tsx#L21)

**Comment by marwan-mohamed12** [comment]
@AugustinMauroy 
I've added the suggested modifications

**Review by AugustinMauroy — COMMENTED** [review]
cloud you rename i18n key and put at correct area in `en.json`
`layouts.blog.subtitle` -> `components.blog.blogHeader.subtitle`
`layouts.blog.blogHeader.RSS` -> ``components.blog.blogHeader.rssLink`

And after that it's will be perfect

**Review by AugustinMauroy — APPROVED** [review]
LGTM !!! Thanks for your contribution 😄

**Comment by marwan-mohamed12** [comment]
> LGTM !!! Thanks for your contribution 😄

Happy to help 😄

**Review by ovflowd — APPROVED** [review]

**Pull request merged**